packages/logger/src/formatter/LogFormatter.ts

```diff
@@ -95,7 +95,7 @@ abstract class LogFormatter implements LogFormatterInterface {
     }
 
     const stackLines = stack.split('\n');
-    const regex = /\((.*):(\d+):(\d+)\)\\?$/;
+    const regex = /\(([^)]*?):(\d+):(\d+)\)\\?$/;
 
```

Comment:
I know this wasn't mentioned in the initial issue, but could you please make the two `\d+` group also lazy by changing them to `\d+?`?

This way they would still match between one and unlimited times but they'd do as few times as possible.